docs/src/model_implementation.md

```diff
@@ -312,5 +314,33 @@ fig
 
 We can see in this that some phytoplankton sink to the bottom, and are both remineralized back into nutrients and stored in the sediment.
 
+### Running on a GPU
+
+In order to run a BGC model on a GPU, the BGC model must first be `adapted`. After the definition of the BGC `struct`, we need to write:
+
+```@example implementing
+using Adapt
+
+import Adapt: adapt_structure
+
+Adapt.adapt_structure(to, bgc::NutrientPhytoplankton) = NutrientPhytoplankton(adapt(to, bgc.base_growth_rate),
+                                                                              adapt(to, bgc.nutrient_half_saturation),
+                                                                              adapt(to, bgc.light_half_saturation),
+                                                                              adapt(to, bgc.temperature_exponent),
+                                                                              adapt(to, bgc.temperature_coefficient),
+                                                                              adapt(to, bgc.optimal_temperature),
+                                                                              adapt(to, bgc.mortality_rate),
+                                                                              adapt(to, bgc.crowding_mortality_rate),
+                                                                              adapt(to, bgc.sinking_velocity))
+```
+
+Also, in order for `ScaleNegativeTracers` to work on a GPU, we must add `grid` as one of the input arguments. We replace the definition of `negative_tracer_scaling` with:
```

Comment:
```suggestion
We can add `grid` as the second argument for `ScaleNegativeTracers` so that it automatically works on a GPU. We replace the definition of `negative_tracer_scaling` with:
```